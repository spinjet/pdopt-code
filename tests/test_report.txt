Total Number of Sets      : 8
Number of Surviving Sets  : 2

Total Surrogate Train Time :        0.428 s
Total Exploration Time     :        0.022 s
Total Search Time          :        1.677 s
Number of Cores Used       :           64

Train time and score of each Surrogate:
       obj    0.1131(s)     1.0000
      con2    0.1730(s)     1.0000
      con1    0.1420(s)     1.0000

Search time and f_evals of each Set:
         2    0.3905(s)
         4    1.2870(s)