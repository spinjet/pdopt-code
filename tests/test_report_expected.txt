Total Number of Sets      : 8
Number of Surviving Sets  : 2

Total Surrogate Train Time :        0.540 s
Total Exploration Time     :        0.021 s
Total Search Time          :        1.649 s
Number of Cores Used       :           64

Train time and score of each Surrogate:
      con2    0.1907(s)     1.0000
      con1    0.2140(s)     1.0000
       obj    0.1350(s)     1.0000

Search time and f_evals of each Set:
         2    0.4081(s)
         4    1.2406(s)